libs/ledger/src/lib/ledger-table.tsx

```diff
@@ -35,24 +37,35 @@ export const TransferTooltipCellComponent = ({
   );
 };
 
+const defaultRangeFilter = { start: formatRFC3339(subDays(Date.now(), 7)) };
+const dateRangeFilterParams = {
+  maxSubDays: 30,
+  maxDaysRange: 7,
```

Comment:
question (non blocking): Why 7 here? From trying out the table it seems a little restrictive. Was there a noticeable increase in time for the query to resolve if you increased it?

Reply:
Additionally, was there a reason for restricting it to 30 days in the past?

If we have a sensible range, I think we should allow any previous date

Reply:
I removed restriction of max 30 days in past.

Reply:
What about the maxDaysRange? I think 7 is is good for the initial query but I don't think we should restrict users from selecting a wider range unless there are performance reasons not too.

Reply:
I'm afraid we will let the users for requesting a tremendous amount of data, hundreds thousands of rows personalised, hard to cache entries and then we will face many timeout errors and high load of servers. I can remove easily period of days limitation - actually it's just remove of one filter parameter, but I wouldn't do it :-|

Reply:
Yeah trying it again and even with 7 days the query takes a long time resolve, not sure if this is causing any danger to the datanode, but lets not risk it for now. We can experiment/discuss with core in future